packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -544,23 +544,25 @@ export class ProjectModel {
         role: ProjectMemberRole,
     ): Promise<void> {
         try {
-            await this.database.raw<
-                (DbProjectMembership & DbProject & DbUser)[]
-            >(
-                `
-                INSERT INTO project_memberships
-                ("role", user_id, project_id)
-                SELECT :role, u.user_id, (
-                    SELECT p.project_id
-                    FROM projects as p
-                    WHERE p.project_uuid = :projectUuid)
-                FROM users as u
-                LEFT JOIN emails ON u.user_id = emails.user_id
-                WHERE emails.email = :email
-                
-                `,
-                { role, projectUuid, email },
-            );
+            const [project] = await this.database('projects')
```

Comment:
raw was not capturing errors, I had to revert it back to the previous implementation